# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libspf2/hotfix-debug.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- libspf2-1.2.10/src/include/spf_log.h.vanilla	2025-06-02 23:50:09.248055153 +0200
+++ libspf2-1.2.10/src/include/spf_log.h	2025-06-02 23:50:32.650182718 +0200
@@ -58,7 +58,7 @@
 void SPF_debugv( const char *file, int line, const char *format, va_list ap ) __attribute__ ((format (printf, 3, 0)));
 
 
-#if defined( __STDC_VERSION__ ) && __STDC_VERSION__ >= 199901L
+#if 0
 
 #define SPF_errorf(format, ... ) SPF_errorx( __FILE__, __LINE__, format, __VA_ARGS__ )
 #define SPF_warningf(format, ... ) SPF_warningx( __FILE__, __LINE__, format, __VA_ARGS__ )
